nixos/modules/services/x11/xserver.nix

```diff
@@ -523,6 +523,16 @@ in
         '';
       };
 
+      logFile = mkOption {
+        type = types.nullOr types.str;
+        # Log at the default verbosity level to stderr rather than /var/log/X.*.log.
```

Comment:
It would be nice to have this in the description instead of a comment ("pass /dev/null to log to stderr").